src/components/forms/FieldTextInput.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';

```

Comment:
ways to import react, react 17 update

https://legacy.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html